tidb-cloud/terraform/backup-resource.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Backup Resource
+summary: Learn how to use backup resource
+---
+
+# Backup Resource
+
+> **Note:**
+>
+> Make sure you have set up a dedicated cluster
```

Comment:
```suggestion
> Because the backup and restore feature is unavailable for Developer Tier clusters. To use backup resources, make sure that you have created a Dedicated Tier cluster. 
```